README.md

```diff
@@ -1,5 +1,9 @@
 # cfgov-refresh
 
+[![Build Status](https://travis-ci.org/cfpb/cfgov-refresh.png?branch=flapjack)](https://travis-ci.org/cfpb/cfgov-refresh?branch=flapjack)
+[![Coverage Status](https://coveralls.io/repos/cfpb/cfgov-refresh/badge.png?branch=flapjack)](https://coveralls.io/r/cfpb/cfgov-refresh?branch=beta)
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/a386709db2a7edff603b2999ebabacbbefb8b4a2 Also updated badge image URL to that shown in code climate example.